README.md

```diff
@@ -1 +1,76 @@
-# modelplace
\ No newline at end of file
+This is an instruction on how to integrate a model to [Modelplace.AI](https://modelplace.ai/).  
+We support [PyTorch](https://pytorch.org/), [Tensorflow](https://www.tensorflow.org/), [OpenVINO](https://docs.openvinotoolkit.org/latest/index.html), and [OnnxRuntime](https://www.onnxruntime.ai/) backends. 
+
+## Publishing information
+For publishing you must provide information about yourself and the model
+### Required
+**Model description** - this is the whole description of what the model does (e.g. [model page](https://modelplace.ai/models/2) - the text in Summary block).
+
+**Model description preview short** - the text people see on a card on the [main page](https://modelplace.ai/) (just after the model name)
+
+**Model mame full** - on a model page - (e.g. Faster region-based convolutional neural network with ResNet-50 FPN backbone https://modelplace.ai/models/2)
+
+**Model name short** - on the [main page](https://modelplace.ai/) (e.g. Faster R-CNN).
+
+**Dataset name** - it's may be public data e.g. MSCOCO or Open images, or your internal dataset.
+
+**Preview image** - we will run a model on the image and will use its output for visualization. This image will be both on the [main page](https://modelplace.ai/) and on the [model page](https://modelplace.ai/models/2)
+
+**Licence** - licence type e.g. Apache 2.0, MIT, Proprietary and so on
```

Comment:
Spelling: License

Reply:
Missed two spaces at the end of the line. Won't be a problem if bullet points is used